src/editor/normal_mode.rs

```diff
@@ -89,7 +92,7 @@ impl<'arena, Node: Ast<'arena>> state::State<'arena, Node> for State {
     }
 
     fn keystroke_buffer(&self) -> Cow<'_, str> {
-        Cow::from(format!("{:?}", self.keystroke_buffer))
+        Cow::from(format!("{}", keystrokes_to_string(&self.keystroke_buffer)))
```

Comment:
I don't think we need to `format!` the string - `Cow::from(keystrokes_to_string(&self.keystroke_buffer))` should work.